Build/ansible/playbooks/ubuntu_14_16_x86.yml

```diff
@@ -135,6 +135,39 @@
             job="/usr/bin/apt-get update && /usr/bin/apt-get -y upgrade"
             state=present
 
+    - name: Install Test tools
+      apt: name={{item}} state=installed
+      with_items:
+        - ant
+        - ant-contrib
+        - git
```

Comment:
Hi @gdams, I didn't change this bit, still just like before, but if you guys think that would be good, I can change that.